src/resolvers/message.ts

```diff
@@ -39,22 +40,22 @@ const resolver: Resolvers = {
         }
 
         const authUsers = [...users, auth.userId];
-        const channel = await channelModel.findOne({
-          include: [
-            {
-              model: membershipModel,
-              as: 'memberships',
-              where: {
-                userId: authUsers,
-              },
-              attributes: [
-                'channelId', 'userId',
-              ],
-            },
+        const memberships = await membershipModel.findAll({
+          attributes: [
+            'channelId',
+            [sequelize.fn('count', sequelize.fn('DISTINCT', sequelize.col('userId'))), 'memberCnt'],
           ],
+          where: {
+            userId: authUsers,
+          },
+          group: ['channelId'],
+          raw: true,
```

Comment:
This "membershipModel.findAll(...)" Sequelize query makes raw query as below.

```
SELECT `channelId`, count(DISTINCT (`userId`)) AS `memberCnt`
FROM `memberships` AS `membership`
WHERE (`membership`.`deletedAt` IS NULL AND
       `membership`.`userId` IN ('c4cfe890-5fc4-11ea-8c3d-c9155d1819cc', '9a59f370-5ba1-11ea-baed-e927eab06e55'))
GROUP BY `channelId`;
```
And as you know, people in a channel are unique. I mean, there are not two same persons in a channel, which means you don't need 'DISTINCT' here since 'DISTINCT' may cause a performance problem.
So, the aggregation clause in 'attribute' could be written as
`[sequelize.fn('COUNT', sequelize.col('userId')), 'memberCnt'],`